zim/plugins/pageindex/__init__.py

```diff
@@ -473,6 +474,11 @@ def do_row_activated(self, treepath, column):
 		model = self.get_model()
 		treeiter = model.get_iter(treepath)
 		mytreeiter = model.get_user_data(treeiter)
+		selected_path = self.get_selected_path()
```

Comment:
`get_selected_path()` gives some overhead vs. a plain  `get_indexpath(mytreeiter)`, but I'd guesstimate its impact to be negligible.